Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1325 @@
+
+
+def list_watchlists_command(client: Client) -> CommandResults:
+
+    contents = []
+    headers = ['ID', 'Name', 'Description', 'create_timestamp', 'Alerts_enabled', 'Tags_enabled', 'Report_ids',
+               'Last_update_timestamp', 'Classifier']
+    result = client.list_watchlists_request()
+    watchlists = result.get('results')
+    if not watchlists:
+        return 'No watchlists were found.'
+
+    for watchlist in watchlists:
+        contents.append({
+            'Name': watchlist.get('name'),
+            'ID': watchlist.get('id'),
+            'Description': watchlist.get('description'),
+            'Tags_enabled': watchlist.get('tags_enabled'),
+            'Alerts_enabled': watchlist.get('alerts_enabled'),
+            'create_timestamp': convert_unix_to_timestamp(watchlist.get('create_timestamp')),
+            'Last_update_timestamp': convert_unix_to_timestamp(watchlist.get('last_update_timestamp')),
+            'Report_ids': watchlist.get('report_ids'),
+            'Classifier': watchlist.get('classifier')
+        })
+
+    readable_output = tableToMarkdown('Watchlists list ', contents, headers, removeNull=True)
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Watchlist',
+        outputs_key_field='id',
+        outputs=contents,
+        readable_output=readable_output,
+        raw_response=result
+    )
+    return results
+
+
+def get_watchlist_by_id_command(client: Client, args: dict) -> CommandResults:
+
+    watchlist_id = args.get('watchlist_id')
+    result = client.get_watchlist_by_id_request(watchlist_id)
+    headers = ['ID', 'Name', 'Description', 'create_timestamp', 'Alerts_enabled', 'Tags_enabled', 'Report_ids',
+               'Last_update_timestamp', 'Classifier']
+
+    contents = {
+        'Name': result.get('name'),
+        'ID': result.get('id'),
+        'Description': result.get('description'),
+        'Tags_enabled': result.get('tags_enabled'),
+        'Alerts_enabled': result.get('alerts_enabled'),
+        'create_timestamp': convert_unix_to_timestamp(result.get('create_timestamp')),
+        'Last_update_timestamp': convert_unix_to_timestamp(result.get('last_update_timestamp')),
+        'Report_ids': result.get('report_ids'),
+        'Classifier': result.get('classifier')
+    }
+
+    readable_output = tableToMarkdown(f'Watchlist {watchlist_id} information', contents, headers, removeNull=True)
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Watchlist',
+        outputs_key_field='id',
+        outputs=contents,
+        readable_output=readable_output,
+        raw_response=result
+    )
+    return results
+
+
+def watchlist_alert_status_command(client: Client, args: dict) -> str:
+    watchlist_id = args.get('watchlist_id')
+    result = client.watchlist_alert_status_request(watchlist_id)
+
+    if not result.get('alert'):
+        return f'Watchlist {watchlist_id} alert status is false'
```

Comment:
that is the terminology in the ui? watchlist status can be true or false?

Reply:
@Itay4 In the UI it's `Alerts:Off` in the API is true/false. 
What do you want to go with?